acceptance/lib/helper.rb

```diff
@@ -196,6 +196,26 @@ def get_defaults_var(host, varname)
     stdout
   end
 
+  # Convert the contents of the certificate file in cert_file on the host
+  # specified by cert_host into an X.509 certificate and return it
+  # cert_host: The host whose cert you want
+  # cert_file: The specific cert file you want
+  # silent   : Suppress Beaker's output; set to false to see it
+  def encode_cert(cert_host, cert_file, silent = true)
+    rawcert = on(cert_host, "cat #{cert_file}", {:silent => silent}).stdout.strip
+    OpenSSL::X509::Certificate.new(rawcert)
+  end
+
+  # Convert the contents of the private key file in key_file on the host
+  # specified by key_host into an RSA private key and return it
+  # cert_host: The host whose cert you want
```

Comment:
@camlow325 ++ Yeah. I'll push a fix. Misleading comments might be worse than incorrect code.